aws/resource_aws_dynamodb_table.go

```diff
@@ -111,6 +111,7 @@ func resourceAwsDynamoDbTable() *schema.Resource {
 			"ttl": {
 				Type:     schema.TypeSet,
 				Optional: true,
+				Computed: true,
```

Comment:
Generally speaking: setting `Computed: true` is not an ideal situation because it prevents Terraform from being able to detect any sort of API drift when an attribute is not defined in the configuration. In this case, people would most likely be expecting that Terraform let's them know when TTL is enabled on a DynamoDB table outside of Terraform when its disabled by default and its not defined in their configuration.

We'll need to figure out how to support this without `Computed: true` most likely. An option might be setting this attribute up like how the `aws_codebuild_project` `cache` attribute is with a `DiffSuppressFunc` and changing `enabled` to `Optional: true` (and potentially `Default: false`).

Reply:
Are the `Computed`, `DiffSuppressFunc`, etc properties documented anywhere?  I could not find them and was not actually sure what `Computed` means.  I didn't know about `DiffSuppressFunc` - I copied this strategy from the `aws_s3_bucket` resource, but could take that approach instead.